src/harness/resources/3d_frag.glsl

```diff
@@ -66,7 +66,6 @@ void main() {
 
         if ((u_material_flags & BR_MATF_LIGHT) != 0u) {
             // TODO: lighting calculations based on https://rr2000.cwaboard.co.uk/R4/BRENDER/TEBK_43.HTM#0
-            uint range = u_material_index_range;
```

Comment:
You can keep it here. The glsl compiler is not spewing warnings.

Reply:
all good, this line was only here because it stopped `u_material_index_range` being optimized away on OSX :)